code/game/objects/structures/crates_lockers/closets/secure/security.dm

```diff
@@ -225,6 +225,7 @@
 	new /obj/item/radio/headset/headset_sec/alt(src)
 	new /obj/item/clothing/glasses/hud/security/sunglasses(src)
 	new /obj/item/flashlight/seclite(src)
+	new /obj/item/card/id/job/security_officer(src)
```

Comment:
This is too much and makes gaining access to security trivial

Reply:
As my layest comment states, the ID is blank and does not include any access. At most it will make your runechat text a cool red.

Reply:
I've been told that deputy IDs work as real IDs and can provide Security Officers with minimal access or for a HoS to get them more access via their department console for Security Officers. However, I feel like this is a bug as anyone can then literally copy and remove your ID if they get a hold of it.